src/index.js

```diff
@@ -1,2 +1,40 @@
-// Import styles
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/225524065-eb282faf-df77-48b6-89be-ba8e02d0f96b.png)

- [ ] It would be nice if you add the following commands in their respective sections to make your readme file more descriptive and professional 😊

## 💻 Getting Started <a name="getting-started"></a>

To get a local copy up and running, follow these steps.

### Prerequisites
```
Git
VS Code
Node
```

### Setup
```
cd my-folder
git clone git@github.com:everinurmind/leaderboard-n.git
cd leaderboard-n
```
### Install
```
npm install
```
### Usage
```
npm run build
npm start
```
### Run tests
```
npm test
```